src/constants.js

```diff
@@ -11,6 +11,7 @@ const DEFAULTS = {
 const DEFAULT_ASPECT_RATIO = '16:9';
 const DEFAULT_IFRAME_HORIZONTAL_PLAYLIST = false;
 const DEFAULT_MAX_WIDTH = '100%';
+const DEAFULT_RESPONSIVE_CLASS = 'responsive-container';
```

Comment:
Typo: `DEAFULT`

Also, all player classes should be prefixed with `vjs-`. This name is too generic and likely to conflict with users' websites.